includes/azure-app-configuration-role-assignment.md

```diff
@@ -18,3 +18,4 @@ Assign the proper App Configuration role assignments to the credentials being us
 1. Select the service principal associated with the service connection that you created in the previous section.
 
     :::image type="content" source="./media/azure-app-configuration-role-assignment/add-role-assignment.png" alt-text="Screenshot shows the Add role assignment dialog.":::
+1. If the store contains Key Vault references, go to relevant Key Vault and assign **Key Vault Secret User** role to the service principal above. Select **Access policies** in the left menu, and ensure **Azure role-based access control** is selected as the permission model.
```

Comment:
Rather than a step, this would be better to have as a Note since not every person going through the docs is required to do this step, only a subgroup. Also, rather than "to the service principal above" it would be better to say "tot he service principal created in the previous step". 
For the following steps, do these take place in the App Config resource or Key Vault's? When I go to key vault I would expect a button that lets me opt into RBAC as the permission model, but I don't see that. Could you make this more clear?

Reply:
I think it's okay for this to be an optional step as long as you are clear about who will need this step at the beginning of the step (like what you have now). I agree with other feedback Alex shared. I would avoid saying something like "in the left" unless I have a screenshot. Ideally, you just talk about it and link to a Key Vault doc, so you don't have to worry about Key Vault changing the UI in the future or the customer wants to do it in CLI, etc.

Reply:
Thank you for pointing that out. Updated to include a link to Key vault doc and updated steps to be more clear.